docs/developer-docs/release-procedure.md

```diff
@@ -0,0 +1,77 @@
+# Release Procedure
+
+This document is a Core team guide, provided publicly to ensure transparancy in the release process.
+
+## Versioning
+
+Jellyfin uses a 3-digit semantic versioning scheme. All releases will have versions in the `X.Y.Z` format, starting from `10.0.0`. Note however that the `10.X.Y` release chain represents the "cleanup" of the codebase, so it should be accepted that `10.X.Y` breaks all compatibility, at some point, with previous Emby-compatible interfaces.
+
+#### `X` - Major breaking versions
+
+The first digit represents a consistent release without breaking 3rd party interfaces including APIs.
+
+This value will be increased if a release:
+
+* Breaks compatibility with the 3rd party API.
+* Breaks compatibility with the plugins API.
+* Makes significant enough changes to warrant a major version bump, as decided by the Core team.
```

Comment:
This line is vague. We should advertise that major makes breaking API changes and minor makes backwards compatible API changes.

We might want to make a major bump if we introduce new endpoints, but maybe not. Let's leave that situation for discussion when it happens and have the docs cover the typical.